test/geocoders/nominatim.py

```diff
@@ -209,18 +215,16 @@ def test_missing_reverse_details(self):
         )
 
     def test_view_box(self):
-        box_tuple = tuple([180, 22, -180, -22])
-        geolocator = Nominatim(view_box=box_tuple)
-        location2 = geolocator.geocode("Peru")
-        self.assertEqual((location2.latitude, location2.longitude),
-                         (-6.8699697, -75.0458515))
-        box_tuple = [Point(22, 180), Point(-22, -180)]
-        geolocator = Nominatim(view_box=box_tuple)
-        location2 = geolocator.geocode("Peru")
-        self.assertEqual((location2.latitude, location2.longitude),
-                         (-6.8699697, -75.0458515))
-        box_tuple = tuple(["180", "22", "-180", "-22"])
-        geolocator = Nominatim(view_box=box_tuple)
-        location2 = geolocator.geocode("Peru")
-        self.assertEqual((location2.latitude, location2.longitude),
-                         (-6.8699697, -75.0458515))
+        without_viewbox = self.geocode_run(
+            {"query": "Maple Street"},
+            {"latitude": 36.809551, "longitude": -97.050604},
+        )
+        for view_box in [(-0.9, 52, -0.11, 50),
```

Comment:
Maybe it's worth expanding the longitudes a little? The result `-0.138` doesn't lie within the `-0.11...-0.9` range. I think something like `-0.15..-0.11` is less likely to accidentally break.